### DISCUSSION TITLE
How to delete activity logs?

### DISCUSSION THREAD

**Discussion by GoDotDotDot** [vercel/community#125] [Help] [answered]
I want to clear the activity logs, whan can I do ?
![image](https://user-images.githubusercontent.com/18232635/137445468-4bb4c00b-1b34-4440-8892-8fd58a2762c8.png)

**Comment by dexdrain** [comment]
Wondering the same

**Comment by Pieparker** [comment, marked as answer]
Hey y'all! For security and auditing purposes, today, there's no such way to delete the activity log. You may wish to consider using a different account or team, if this is a big concern for you!
